plugins/youtube/plugin.py

```diff
@@ -118,8 +119,8 @@ def _parse_item(self, item):
         uploader = str(uploader.encode('utf-8'))
         video_id = str(video_id.encode('utf-8'))
 
-        return ("[ Title: %s | Uploaded by: %s | https://www.youtube.com/watch?v=%s ]" %
-            (title, uploader, video_id))
+        return ("[ Title: %s | Uploaded by: %s | %s Views | https://www.youtube.com/watch?v=%s ]" %
```

Comment:
I'll update this to "Views: %s" to be more in line with the other data in this message unless you see a reason I shouldn't.